.eslintrc.json

```diff
@@ -0,0 +1,121 @@
+    "curly": 2,
+    "default-case": 2,
+    "no-fallthrough": 2,
+    "dot-location": [2, "property"],
+    "dot-notation": 2,
+    "eqeqeq": [2, "allow-null"],
+    "no-alert": 2,
+    "no-caller": 2,
+    "no-case-declarations": 2,
+    "no-else-return": 2,
+    "no-extra-label": 2,
+    "no-empty-pattern": 2,
+    "no-eval": 2,
+    "no-implied-eval": 2,
+    "no-extend-native": 2,
+    "no-extra-bind": 2,
+    "no-floating-decimal": 2,
+    "no-iterator": 2,
+    "no-lone-blocks": 2,
+    "no-loop-func": 2,
+    "no-multi-spaces": 2,
+    "no-multi-str": 2,
+    "no-native-reassign": 2,
+    "no-new-func": 2,
+    "no-new-wrappers": 2,
+    "no-new": 2,
+    "no-octal-escape": 2,
+    "no-proto": 2,
+    "no-script-url": 2,
+    "no-self-compare": 2,
+    "no-sequences": 2,
+    "no-throw-literal": 2,
+    "no-unused-expressions": [ 2, { "allowShortCircuit": true }],
+    "no-unused-vars": [ 2, { "argsIgnorePattern": "^_" }],
+    "no-useless-call": 2,
+    "no-useless-concat": 2,
+    "no-void": 2,
+    "no-with": 2,
+    "radix": [ 2, "as-needed"],
+    "wrap-iife": [2, "any"],
+    "strict": [2, "safe"],
+    "no-shadow": 1,
+    "no-shadow-restricted-names": 1,
+    "no-undef": 2,
+    "no-use-before-define": [ 2, { "functions": false }],
+    "no-console": 0,
+
+    // style stuff
+    "block-spacing": 2,
+    "brace-style": ["error", "1tbs", { "allowSingleLine": true }],
+    "camelcase": 2,
+    "comma-dangle": 0,
+    "comma-spacing": 2,
+    "comma-style": 2,
+    "computed-property-spacing": 2,
+    "eol-last": 2,
+    "no-trailing-spaces": 2,
+    "indent": [2, 2, { "SwitchCase": 1 }],
+    "key-spacing": 2,
+    "linebreak-style": 2,
+    "spaced-comment": 2,
+    "max-depth": [ 1, 3 ],
+    "max-len": [ 2, 80, 2 ],
+    "max-nested-callbacks": [ 1, 5 ],
+    "no-array-constructor": 2,
+    "no-bitwise": 2,
+    "no-lonely-if": 2,
+    "no-multiple-empty-lines": [2, { "max": 2 }],
+    "no-negated-condition": 2,
+    "no-nested-ternary": 2,
+    "no-unneeded-ternary": 2,
+    "no-new-object": 2,
+    "no-spaced-func": 2,
+    "object-curly-spacing": [ 2, "always", { "objectsInObjects": false, "arraysInObjects": true }],
+    "operator-linebreak": 1,
+    "quote-props": [ 2, "as-needed" ],
+    "semi-spacing": 2,
+    "semi": 2,
+    "keyword-spacing": [2, { "overrides": {
+      "switch": { "after": false }
+    }}],
+    "space-before-blocks": 2,
+    "space-before-function-paren": [ 2, "never"],
+    "space-in-parens": 2,
+    "space-infix-ops": 2,
+    "space-unary-ops": 2,
+
+    // es6 stuff
+    "arrow-body-style": ["warn", "as-needed"],
+    "arrow-spacing": 2,
+    "constructor-super": 2,
+    "no-confusing-arrow": [2, { "allowParens": true }],
+    "no-constant-condition": 2,
+    "no-class-assign": 2,
+    "no-const-assign": 2,
+    "no-dupe-class-members": 2,
+    "no-this-before-super": 2,
+    "no-var": 2,
+    "prefer-const": 2,
+    "prefer-spread": 2,
+    "prefer-template": 2,
+    "require-yield": 2
+  }
+}
```

Comment:
actually I took it from calendar-server, but you're right I should have taken it from the former app.